### DISCUSSION TITLE
Forking a repo twice

### DISCUSSION THREAD

**Discussion by HPCguy** [community/community#23095] [New to GitHub]
<p>I have a fork of rswier/c4 in my account and would also like to have a fork or EarlGray/c4, which was originally forked from rswier/c4, but has changed so dramatically, that they are different repos at this point, and EarlGray/c4 stopped pulling from rswier/c4 long ago.</p>
<p>I tried this:<br>
% Created a repo, HPCguy/EarlGray_c4<br>
% git clone --bare <a href="https://github.com/EarlGray/c4.git" class="inline-onebox" rel="noopener nofollow ugc">GitHub - EarlGray/c4: x86 JIT compiler in 86 lines</a><br>
% cd c4.git<br>
% git push --mirror <a href="https://github.com/HPCguy/EarlGray_c4/git" rel="noopener nofollow ugc">https://github.com/HPCguy/EarlGray_c4/git</a></p>
<p>But under no circumstance could I force github to accept it.</p>
<p>Really, github, this should be supported when two fors of a repo diverge so dramatically.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>my suggestion is for you to create an Org, even if you don’t have members and fork it there, that way you have two existing forks from the same source</p>
<p>it’s also the suggestion here</p>
<aside class="onebox stackexchange" data-onebox-src="https://stackoverflow.com/questions/12338240/how-can-i-fork-the-original-repo-when-ive-already-forked-a-different-fork">
  <header class="source">

      <a href="https://stackoverflow.com/questions/12338240/how-can-i-fork-the-original-repo-when-ive-already-forked-a-different-fork" target="_blank" rel="noopener">stackoverflow.com</a>
  </header>

  <article class="onebox-body">
      <a href="https://stackoverflow.com/users/157247/t-j-crowder" target="_blank" rel="noopener">
    <img alt="T.J. Crowder" src="https://user-images.githubusercontent.com/47092464/181074827-e531855c-6a3f-4f68-9285-bda99a25aae0.jpeg" class="thumbnail onebox-avatar" width="256" height="256">
  </a>

<h4>
  <a href="https://stackoverflow.com/questions/12338240/how-can-i-fork-the-original-repo-when-ive-already-forked-a-different-fork" target="_blank" rel="noopener">How can I fork the original repo when I've already forked a different fork</a>
</h4>

<div class="tags">
  <strong>github, git-fork</strong>
</div>

<div class="date">
  asked by
  
  <a href="https://stackoverflow.com/users/157247/t-j-crowder" target="_blank" rel="noopener">
    T.J. Crowder
  </a>
  on <a href="https://stackoverflow.com/questions/12338240/how-can-i-fork-the-original-repo-when-ive-already-forked-a-different-fork" target="_blank" rel="noopener">10:16AM - 09 Sep 12 UTC</a>
</div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
